VerifyBamID/2.0.1--h32f71e1_0/CheckContamination.nf

```diff
@@ -0,0 +1,64 @@
+process CheckContamination {
+    tag {"VerifyBamID CheckContamination ${sample_id}.${int_tag}"}
+    label 'VERIFYBAMID_2_0_1_h32f71e1_0'
+    label 'VERIFYBAMID_2_0_1_h32f71e1_0_CheckContamination'
+    container = 'quay.io/biocontainers/verifybamid2:2.0.1--h32f71e1_0'
+    shell = ['/bin/bash', '-eo', 'pipefail']
+
+    input:
+        tuple (
+            sample_id, 
+            path(bam), 
+            path(bai))
+
+    output:
+        tuple (
+            sample_id, 
+            path("${output_prefix}.selfSM"), 
+            stdout)
+
+    script:
+        output_prefix = "${sample_id}.${params.library_strategy}.contamination"
```

Comment:
Why do you add `library_strategy`? I think sample_name (and 'contamination') is sufficient as output_prefix?

Reply:
Haha, yes old habit. It depends on how we store the data and depending on that whether it is clearly separated between WES and WGS (in future). 
I can definitely remove it.